tests/test_queuing.py

```diff
@@ -12,33 +12,52 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """
-Unit tests for the :mod:`pennylane` :class:`QueuingContext` class.
+Unit tests for the :mod:`pennylane` :class:`QueuingManager` class.
 """
 import contextlib
 
 import pytest
 import pennylane as qml
 import numpy as np
 
-from pennylane.queuing import AnnotatedQueue, AnnotatedQueue, QueuingContext, QueuingError
+from pennylane.queuing import AnnotatedQueue, AnnotatedQueue, QueuingManager, QueuingError
 
 
-class TestQueuingContext:
-    """Test the logic associated with the QueuingContext class."""
+def test_top_level_import_warning():
```

Comment:
For some reason this test fails.

Reply:
I was going to stop importing the object top-level, but then I realized I wanted easy access to `qml.QueuingManager.active_context()` and `qml.QueuingManger.stop_recording()`.  So it is once again imported top-level.